cmd/install/assets/hypershift_operator.go

```diff
@@ -276,8 +277,22 @@ func (o HyperShiftOperatorDeployment) Build() *appsv1.Deployment {
 		fmt.Sprintf("--enable-ci-debug-output=%t", o.EnableCIDebugOutput),
 		fmt.Sprintf("--private-platform=%s", o.PrivatePlatform),
 	}
-	var oidcVolumeMount []corev1.VolumeMount
-	var oidcVolumeCred []corev1.Volume
+	volumeMounts := []corev1.VolumeMount{
```

Comment:
Reviewer note: the addition of this `serving-cert` Volume and VolumeMount is the only material change here.  The rest is just a cleanup and no other functional change is intended.

Reply:
Not sure how much we want to care about that, but in non-ocp this won't work since there isn't a controller to generate the serving cert off of the service

Reply:
Yeah... so the annotation on the service that won't be honored isn't a big deal but the secret not existing is a problem.  Might need another Capabilities check here before adding the cert-dir stuff.

Reply:
I believe this should only be enabled if specifying the flag (we will not want that in our deployment)